### PR TITLE
Added farcaster-v2 subgraph

### PR DESCRIPTION
**Pull request by 0xsarvesh**

### CODE REVIEW

File: farcaster_v2/src/id-registry.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+): void {
+
+  airstack.social.trackSocialUserRecoveryAddressChangeTransaction(
+    event.block,
+    event.transaction.hash.toHexString(),
+    event.logIndex,
+    event.transaction.from.toHexString(),
+    event.address.toHexString(),
+    event.params.id.toString(),
+    FARCASTER_ID_REGISTRY_CONTRACT,
```
Comment (sharathkrml) [inline farcaster_v2/src/id-registry.ts:79]: we can use `event.address` also rather than a hardcoded value

**Review by sharathkrml — DISMISSED** [review]
LGTM, test cases are missing

File: farcaster_v2/src/id-registry.ts (hunk 1 above)
Comment (0xsarvesh) [reply farcaster_v2/src/id-registry.ts:86]: Done

**Review by sharathkrml — APPROVED** [review]

File: farcaster_v2/src/id-registry.ts (hunk 1 above)
Comment (sharathkrml) [reply farcaster_v2/src/id-registry.ts:86]: Seems like changes are made at one place(line 69),but not at (line 86)

**Pull request merged**